examples/create-machine.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const Machine = require('..');
```

Comment:
Can you remove the semicolons here and below (11x)? Thanks!

Reply:
Done!